### DISCUSSION TITLE
Harman AIR Website - Making "Discussions" (=Community) more visible / present?

### DISCUSSION THREAD

**Discussion by 2jfw** [airsdk/Adobe-Runtime-Support#1156] [Ideas]
Like the title says - currently, official website "https://airsdk.harman.com/" does only link to that page discretely so why not emphasizing the **Community** and **Discussions** here more?

The community is a very important part of the as3(/flash) eco system so highlighting and advertising it could help.

I am just thinking of one of the last bigger STEAM updates where Valve changed the UI in way to more focus on and point the Gamers to the community/forums and animating them the write reviews etc..  **EDIT:** .. and that **DID** actually work out well (I just dont find the source for that anymore..)

**Comment by ajwfrost** [comment]
Good idea! .... that page is under our main domain and we've been required to use the corporate style guide etc which is somewhat constraining. Pointing to these pages under the github/airsdk account - and to the associated website - is a very good idea.

As a community-driven space, I'm hoping that we'll get more contributions to the https://airsdk.dev/ website - and the first step (which we were about to take anyway!) is to link to it from the Harman website... This will then be more dynamic, and the Harman-domain website would just be for the actual releases/deliveries etc.

**Reply by 2jfw** [reply]
> Good idea! .... that page is under our main domain and we've been required to use the corporate style guide etc which is somewhat constraining. Pointing to these pages under the github/airsdk account - and to the associated website - is a very good idea.
> 
> As a community-driven space, I'm hoping that we'll get more contributions to the https://airsdk.dev/ website - and the first step (which we were about to take anyway!) is to link to it from the Harman website... This will then be more dynamic, and the Harman-domain website would just be for the actual releases/deliveries etc.

Sounds awesome 😎

Btw, never knew the website https://airsdk.dev existed... looks great and quite up-to-date regarding look & feel 👍
Was it actually announced anywhere?

**Reply by ajwfrost** [reply]
It's a recent thing -> with a huge amount of thanks to @marchbold  ... just about ready to be announced, especially as I've been pointing a lot of folk to the useful article about App Bundles :-)

**Reply by 2jfw** [reply]
Yeah, just saw that Michael was at it again.. I mean, who else?! ... Thanks @marchbold  🍻👍

**Reply by marchbold** [reply]
Harman and @ajwfrost  should take the credit here, we would all be doing something else without them! :)

**Comment by ajwfrost** [comment]
@2jfw just wondering if you think the new home page at https://airsdk.harman.com is any better at promoting this site and the airsdk.dev site? We reworked the front page a little. Still perhaps a little wordy (designed by engineers....) so all feedback is useful!

**Comment by 2jfw** [comment]
@ajwfrost Slightly better ... I actually thought about linking "Community" or "Discussions" in the top/main navigation (next to News e.g.) as the links in the floating text are not very present/promoted.
Same for "airsdk.dev". I currently have no name for that link - maybe "AIR Home" ?

**Reply by marchbold** [reply]
@2jfw Totally agree, still buried as a text link. 

Regarding wording, I would suggest something like the "Developer Portal" or "Developer Documentation" or even just "Documentation"  for the airsdk.dev site?

**Reply by 2jfw** [reply]
> @2jfw Totally agree, still buried as a text link.
> 
> Regarding wording, I would suggest something like the "Developer Portal" or "Developer Documentation" or even just "Documentation" for the airsdk.dev site?

I think "**Developer Portal**" would suite best, since "Docs" is already a sub-category on airsdk.dev and maybe airsdk.dev will feature more than just docs (like a Showcase maybe ;)). Also seeing there is a "News" section already placeholding...

**Reply by 2jfw** [reply]
By the way, I also feel like airsdk.dev should point to "Discussions" here in the header links.
Its fine when all three pages cross-link each other in the same way since the main parts _are_ spread..

**Reply by marchbold** [reply]
@2jfw I have been discussing putting showcase section on airsdk.dev along with community links with Andrew so will make this happen very soon.

**Reply by marchbold** [reply]
Showcase and community sections are now live on airsdk.dev:

- [Showcase](https://airsdk.dev/showcase)
- [Community](https://airsdk.dev/community)

If you want to put an application up there you can follow the guide to do a PR or just shoot me the details on your app along with an image for the site.

**Reply by 2jfw** [reply]
Looks really great! Loving it..! Sooner or later there may be the necessity to have categories in the showcase to differentiate between mobile and desktop apps, maybe platform, and maybe also tools and games - but for now this all is just perfectly fine 👍
**EDIT:** Oh, I see there are already icons showing the category - sweet! 

I think there should be some kind of announcement of the new show case (and website) in the Starling Forums, too, so we can reach out to as many people as possible to invite them to contribute to it.
An announcement in the Adobe AIR (/Animate) forums could also push the site!

**Reply by marchbold** [reply]
Yep, [tags](https://github.com/airsdk/airsdk.dev/blob/main/src/data/tags.tsx) are already in there, will put a UI together to filter them once we have some more apps in there.  Welcome suggestions on any other tags that we should add too...

**Reply by marchbold** [reply]
(And I need to clean up those icons... )

**Comment by 2jfw** [comment]
Small typo here: "Highlight: _Amazong_ applications that you should check-out!" (APM)

Not sure if already like this (since there aren't much apps yet) but I think we need a separation between "**Utility**" and "**Tool**". Tool being something bigger here.

**Reply by marchbold** [reply]
Oh dear.. yeah "Amazong!" ... 

What would be a "Tool" ?

**Reply by 2jfw** [reply]
A tool would be for example the application we@Dallmeier are working on (and hoping to add to the showcase soon, too) **SeMSy 5** (Security Management): https://www.dallmeier.com/technology/software - see first video e.g.
I would also consider **Moonshine IDE** as a Tool (https://github.com/Moonshine-IDE)

**Reply by marchbold** [reply]
What would you consider a utility then?

**Reply by 2jfw** [reply]
Your AIR Package Manager - as it currently already is flagged as a Utility. However, it may also be called a tool. It's not easy to draw the line between **Tool** and **Utility**. I would consider Utilities to be "smaller" than Tools, maybe with one certain specific purpose only

**Reply by marchbold** [reply]
Yeah okay, might be a fine line there and very much open to interpretation.  Prefer to keep it as clear as possible.

**Reply by 2jfw** [reply]
Sure! For now, it is all fine the way it is!

**Reply by 2jfw** [reply]
(https://english.stackexchange.com/questions/80384/what-s-the-difference-between-tool-and-utility)
